sweep/sweeper_test.go

```diff
@@ -402,9 +399,10 @@ func TestNegativeInput(t *testing.T) {
 	ctx.backend.mine()
 
 	ctx.expectResult(secondLargeInputResult, nil)
-	ctx.expectResult(negInputResult, nil)
 
-	ctx.finish(1)
+	// The negative input's waitForSpend goroutine should still be lingering
+	// since it hasn't been swept.
+	ctx.finish(2)
```

Comment:
If this commit is fixing a failing test (?), I think it should be grouped with the code that introduces the failure.

Reply:
I opted for fixing the test in a separate commit to make it more clear as to why the change was needed. FWIW, it still compiles, so it only fails if you try to run it.

Reply:
Every commit builds is :2nd_place_medal: but getting every commit pass tests is :1st_place_medal:  At least that is what I do :)

Reply:
Done.